iceberg/iceberg-rest-server/src/test/java/org/apache/gravitino/iceberg/service/rest/IcebergRestTestUtil.java

```diff
@@ -119,4 +124,11 @@ protected void configure() {
     }
     return resourceConfig;
   }
+
+  static HttpServletRequest createMockHttpRequest() {
+    HttpServletRequest mockRequest = mock(HttpServletRequest.class);
+    when(mockRequest.getRemoteAddr()).thenReturn("127.0.0.1");
+    when(mockRequest.getHeaderNames()).thenReturn(Collections.emptyEnumeration());
+    return mockRequest;
+  }
```

Comment:
Can you please add several UTs to cover your change for `Context` and various `events`.

Reply:
Updated UT